docs/getting-started.md

```diff
@@ -0,0 +1,161 @@
+- Plug the AC power cable to the socket labelled **POWER IN**, and then to the mains outlet
+- The device should boot up automatically; wait until you see the Etcher interface show up
+- [Warning] Please avoid using adaptors or extension leads, as this may damage the device or cause it to malfunction
+
+### Connecting to WiFi
+
+- The device will prompt you to connect to a local network the first time it boots (you can skip this step if you are not connecting to WiFi)
+- Select the network to which you want to connect, type the password and select **OK**
+- You can access the WiFi settings by selecting the WiFi icon at the top left corner of the screen
+
+### 2.2 Etcher functions
+
+### Flash from file
+
+Using an image file as source to flash to one or multiple targets
+
+- From the Etcher menu, select **Flash from file**
+- Plug a drive that contains the image you would like to flash into the slot with the blue-colored blinking LED
+- Select the image you want to flash from the file browser and select **OK**
+- Plug at least one drive or device into an available slot. The plugged target(s) will be selected automatically and the LEDs will turn white
+- Select **Flash**, to begin the flashing process
+- The LED of each slot will first blink purple for flashing and then green for validating
+- Once the flashing is complete, the LEDs will turn green for successfully flashed drives, and red for the failed ones
+- You may safely unplug the drives when flashing is complete
+
+### Flash from URL
+
+Using an online image file as source to flash to one or multiple targets
+
+- From the Etcher menu, select **Flash from URL**
+- Enter the image URL of you would like to flash in the input field, and select **OK**
+- Plug at least one drive or device into an available slot. The plugged target(s) will be selected automatically, and the LEDs will turn white
+- Select **Flash**, to begin the flashing process
+- The LED of each slot will first blink purple for flashing and then green for validating
+- Once the flashing is complete, the LEDs will turn green for successfully flashed drives, and red for the failed ones
+- You may safely unplug the drives when flashing is complete
+
+### Clone drive
+
+Using a drive as source, and cloning it to multiple drives
+
+- From the Etcher menu, select **Clone drive**
+- Plug a drive you would like to clone into the slot with the blue-colored blinking LED
+- The drive will be selected automatically and the LED will stop blinking
+- Plug at least one drive into an available slot. The plugged targets will be selected automatically, and the LEDs will turn white
+- Select **Flash**, to begin the flashing process
+- The LED of each slot will first blink purple for flashing and then green for validating
+- Once the flashing is complete, the LEDs will turn green for successfully flashed drives, and red for the failed ones
+- You may safely unplug the drives when flashing is complete
+
+### Backup drive
```

Comment:
this section also needs to be removed, sorry, i missed it the first time around. @vipulgupta2048

Reply:
Cool